README.md

```diff
@@ -22,36 +61,40 @@ Next include the base HTML of the InfoBar anywhere inside the HTML of your docum
 </div>
 ```
 
-There is an imporant attribute missing from the above, and without it, the InfoBar will not work. The attribute in question here is the `data-infobar` attribute. This attribute can have one of two values, `update` or `translate` as follows:
+There is an imporant attribute missing from the above, and without it, the InfoBar will not work. The attribute in question here is the `data-infobar` attribute. This attribute can have one, or both, of two possible values, `update` and `translate`. For example:
 
 ``` html
 <div ... data-infobar="update">
-....
 ```
 
-The above will trigger the Update bar and will be shown if the current user is using a version of Firefox that is two major versions older than the latest.
+The above will trigger the Update Bar to be shown if the current user is using a version of Firefox that is two major versions older than the latest release version.
 
-If the value above is translate, it will show the Translation Bar based in the precense of some additional elements as described later. You can also specify both, as such:
+If the value above is translate, it will show the Translation Bar based on the precense of some additional elements as described later. You can also specify both, as such:
 
 ``` html
 <div ... data-infobar="update translate">
-....
 ```
 
-The order here is important, as wo InfoBar elements will never be shown simultaneously. In the scenario above, the first time the page is loaded, the logic for the update bar will be run. Once the user has taken the required action or, dismissed the notification, the translate bar's logic will b run on subsequent page loads.
+The order here is important, as both InfoBar elements will never be shown simultaneously. In the scenario above, the first time the page is loaded, the update bar will be run. Once the user has taken either of the available actions, the Translation Bar will be shown on subsequent page loads.
```

Comment:
Suggest we stick to using `Update Bar` and `Translation Bar` using title case for consistency.